src/bitcoin-util.cpp

```diff
@@ -15,6 +15,7 @@
 #include <streams.h>
 #include <util/system.h>
 #include <util/translation.h>
+#include <version.h>
```

Comment:
Looks like an unrelated change snuck in here?

Reply:
No, the changes here wouldn't compile without this change. 

Commit dbcb5742c48fd26f77e500291d7083e12eec741b removed `INIT_PROTO_VERSION` from protocol.h, but didn't remove the version.h include from protocol, so I am doing that here.

Reply:
Thanks, makes sense.